at_boundaries: {
  room_1: [[0.0, -3.0], [0.0, 3.0], [-6.0, 3.0], [-6.0, -3.0]],
  room_2: [[0.0, -3.0], [0.0, 3.0], [6.0, -3.0], [6.0, 3.0]]
}